master/internal/api_command.go

```diff
@@ -153,7 +153,7 @@ func (a *apiServer) getCommandLaunchParams(ctx context.Context, req *protoComman
 	// Check submitted config against task config policies.
 	valid, err := configpolicy.CheckNTSCConstraints(ctx, int(cmdSpec.Metadata.WorkspaceID), config, a.m.rm)
 	if !valid {
-		return nil, nil, err
+		return nil, nil, status.Errorf(codes.InvalidArgument, "failed constraint check: %v", err)
```

Comment:
@amandavialva01 I had `%w` here at first, but the formatting was super weird. I was reminded of the conversation we had about `%v` vs. `%w` in `fmt.Errorf`. Eventually I'll (hopefully) keep it straight! 😆

Reply:
Loll same! 😄 It's definitely a matter of trial and error 😆